contracts/evoting/controller/handlers.go

```diff
@@ -36,27 +38,27 @@ func (h *votingProxy) Login(w http.ResponseWriter, r *http.Request) {
 
 // CreateElection allows creating an election.
 func (h *votingProxy) CreateElection(w http.ResponseWriter, r *http.Request) {
-	createElectionRequest := &types.CreateElectionRequest{}
-	err := json.NewDecoder(r.Body).Decode(createElectionRequest)
+	req := &types.CreateElectionRequest{}
+
+	err := json.NewDecoder(r.Body).Decode(req)
 	if err != nil {
 		http.Error(w, "failed to decode CreateElectionRequest: "+err.Error(),
 			http.StatusBadRequest)
 		return
 	}
 
-	createElectionTransaction := types.CreateElectionTransaction{
-		Configuration: createElectionRequest.Configuration,
-		AdminID:       createElectionRequest.AdminID,
+	createElection := types.CreateElection{
+		Configuration: req.Configuration,
+		AdminID:       req.AdminID,
 	}
 
-	payload, err := json.Marshal(createElectionTransaction)
+	data, err := createElection.Serialize(h.context)
 	if err != nil {
 		http.Error(w, "failed to marshal CreateElectionTransaction: "+err.Error(),
 			http.StatusInternalServerError)
-		return
```

Comment:
Seems like a typo, why not return after the error ?

Reply:
Ooop true. Thanks!